documentation/docs/libraries/nodejs/how_to/_run_how_tos.mdx

```diff
@@ -36,7 +41,7 @@ npm run build
 The wallet.rs library has numerous examples you can run to get acquainted with the library.
 After you have followed the instructions to [build the library](#build-the-library), you can run any Node.js example.
 You will need to select any of the
-[Node.js examples](https://github.com/iotaledger/wallet.rs/tree/develop/bindings/nodejs/examples) and run it with
+[Node.js examples](https://github.com/iotaledger/wallet.rs/tree/develop/wallet/bindings/nodejs/examples/) and run it with
 your local Node.js interpreter.
 
 To run examples, you will need to change to the `bindings/nodejs/examples` directory, and run any example.
```

Comment:
First need to run `npm i` in the examples folder (David is checking if there is a better solution, but that's how it works now)